src/index.html

```diff
@@ -1,22 +1,204 @@
 <!doctype html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1.0"
-    />
-    <title>Landing Page</title>
-    <link
-      rel="stylesheet"
-      href="./styles/main.scss"
-    />
-    <script
-      src="scripts/main.js"
-      defer
-    ></script>
-  </head>
-  <body>
-    <h1>Landing Page</h1>
-  </body>
+
+
+<head>
+
```

Comment:
You should add the logo as a favicon [Reference](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md#:~:text=Use%20the%20logo%20as%20a%20favicon)